src/pages/docs/tutorials/setup/index.md

```diff
@@ -94,7 +94,10 @@ Your application is configured using the `package.json` file. We'll only cover t
 		"theme": "sandstone"
 	},
 	"eslintConfig": {
-		"extends": "enact"
+		"extends": "enact",
+		"ri": {
```

Comment:
Why `ri` is here in the `eslintConfig` block?

Reply:
I'm not sure we need to change this without a guide since the baseSize of sandstone if `48` not  `24`.